data-processing/scripts/miljoeportal/README.md

```diff
@@ -0,0 +1,7 @@
+# Danmarks Miljøportal
+
+To pull data run:
+
+```bash
+$ uv run data-processing/scripts/miljoeportal/download.py $PATH_TO_SAVE_RAW_DATA $PATH_TO_SAVE_EXTRACTED_DATA4
```

Comment:
```suggestion
$ python data-processing/scripts/miljoeportal/download.py $PATH_TO_SAVE_RAW_DATA $PATH_TO_SAVE_EXTRACTED_DATA4
```
I think we should be consistent throughout (I am perfectly fine with consistently using `uv`